README.md

```diff
@@ -69,70 +70,72 @@ Follow the steps below to clone the API-M documentation GitHub repository and to
 1. Fork the GitHub repository: `https://github.com/wso2/docs-apim.git`
 2. Navigate to the place where you want to clone the repo.
 
-    Git clone the forked repository.
+   Git clone the forked repository.
 
-    ```shell
-    $ git clone https://github.com/[git-username]/docs-apim.git
-    ```
+   ```shell
+   $ git clone https://github.com/[git-username]/docs-apim.git
+   ```
 
 3. Navigate to the folder containing the repo that you cloned in step 3.2 on a terminal window.
 
-    For example:
+   For example:
 
-    ```shell
-    $ cd docs-apim/<Language-folder>/
-    ```
+   ```shell
+   $ cd docs-apim/<Language-folder>/
+   ```
 
-    ```shell
-    $ cd docs-apim/en/
-    ```
+   ```shell
+   $ cd docs-apim/en/
+   ```
 
 4. Install the required pip packages.
 
-    This will install MkDocs and the required theme, extensions, and plugins.
+   This will install MkDocs and the required theme, extensions, and plugins.
 
-    - If you are using Python2, use the following command:
+   - If you are using Python2, use the following command:
 
-      ```shell
-      $ pip install -r requirements.txt
-      ```
+     ```shell
+     $ pip install -r requirements.txt
+     ```
 
-    - If you are using Python3, use the following command:
+   - If you are using Python3, use the following command:
 
-      ```shell
-      $ pip3 install -r requirements.txt
-      ```
+     ```shell
+     $ pip3 install -r requirements.txt
+     ```
 
 ### Step 4 - Run MkDocs
+
 1. Run the following command to start the server and view the site on your local server.
 
-    ```shell
-    $ python3 -m mkdocs serve
-    ```
-
-    > **NOTE:**
-    >
-    > If you are making changes and want to see them on the fly, run the following command to start the server and view the site on your local server.
-    > 1. Navigate to the `mkdocs.yml` file.
-    > 2. Change the following configuration to `false` as shown below. 
-    >     ```
-    >     #Breaks build if there's a warning
-    >     strict: false
-    >     ```
-    > 3. Run the following command to start the server and to make the server load only the changed items and display the changes faster. 
-    >
-    >    `python3 -m mkdocs serve --dirtyreload`
-  
+   ```shell
```

Comment:
Couldn't see a difference in rendering the readme.md with this change of indentation. Will you be able to check?